R/learner_BART_surv_bart.R

```diff
@@ -0,0 +1,293 @@
+#' @title Survival Bayesian Additive Regression Trees Learner
+#' @author bblodfon
+#' @name mlr_learners_surv.bart
+#'
+#' @description
+#' Fits a Bayesian Additive Regression Trees (BART) learner to right-censored
+#' survival data.
+#' For prediction, we return the mean posterior estimates of the survival
+#' function and the corresponding `crank` (expected mortality) using
+#' [mlr3proba::.surv_return].
+#' The full posterior estimates are currently stored in the
+#' `learner$state$surv_test` slot, along with the number of test observations
+#' `N`, number of unique times in the train set `K` and number of posterior
+#' draws `M`.
+#' See example for more details.
+#'
+#' Calls [BART::mc.surv.bart()] from \CRANpkg{BART}.
+#' Based on BART version 2.9.4 (2023-03-25).
+#'
+#' @section Custom mlr3 defaults:
```

Comment:
This section should be called "Initial parameter values" (it was wrong in the tutorial, I already opened a PR to fix this).